packages/rx_bloc_cli/mason_templates/bricks/lib_internationalisation/__brick__/lib/lib_internationalisation/data_sources/language_local_data_source.dart

```diff
@@ -0,0 +1,66 @@
+{{> licence.dart }}
+
+import 'package:widget_toolkit/language_picker.dart';
+
+import '../data_storages/language_picker_shared_preferences_instance.dart';
+import '../utils/language_util.dart';
+
+/// Example for a language data source implementation
+class LanguageLocalDataSource {
+  static const keyCurrent = 'languageCurrent';
+  static const keyBG = 'languageCurrentBG';
+  static const keyEN = 'languageCurrentEN';
+  static const showErrorValue = 'showError';
+
+  LanguageLocalDataSource(this._storage);
+
+  final LanguagePickerSharedPreferencesInstance _storage;
```

Comment:
Let's reduce the complexity as injecting the generic SharedPreferences data source here

Reply:
`LanguagePickerSharedPreferencesInstance ` was replaced with `SharedPreferencesInstance `